libs/tasks/feature/src/lib/tasks-feature/tasks-feature.component.html

```diff
@@ -1,21 +1,27 @@
-<todo-workspace-tasks-list [tasks]='tasks$ | complete:false | async'>
-  <h1>TODO List</h1>
-  <button mat-raised-button color="primary" (click)='openCreateDialog()'>New task</button>
-</todo-workspace-tasks-list>
-<!--<mat-paginator [length]='length'-->
-<!--               [pageSize]='pageSize'-->
-<!--               [pageSizeOptions]='pageSizeOptions'-->
-<!--               (page)='pageEvent = $event'>-->
-<!--</mat-paginator>-->
+<div class='task-list-container'>
+  <todo-workspace-tasks-list class='task-list-container__task-list  mat-card mat-elevation-z3'
+                             (deleteEvent)='tasksFacade.delete($event._id)'
+                             (editEvent)='openEditDialog($event)'
+                             (changePage)='tasksFacade.changePageTodo($event.event, $event.tasksNumber)'
+                             (completionChangeEvent)='tasksFacade.changeCompletion($event)'
+                             [tasks]='tasks | complete: false | artificialPaginator: todoPage$ | async'
+                             [pagination]='todoPage$ | async'>
```

Comment:
then you don't need to use async. If you used  property `as todoPage` now you can provide here `todoPage` without `async`